client/js/components/GameView/GameDialogs.vue

```diff
@@ -124,6 +127,14 @@ export default {
     stalemate() {
       return this.$store.state.game.gameIsOver && this.$store.state.game.winnerPNum === null;
     },
+    consideringOpponentStalemateRequest: {
+      get() {
+        return this.$store.state.game.consideringOpponentStalemateRequest;
+      },
+      set(val) {
+        this.$store.commit('setConsideringOpponentStalemateRequest', val);
+      },
+    },
```

Comment:
I mulled over the best way to do this for a while when I was breaking these components out. But due to needing to set and get from the same value for Vuetify components I landed on this just being acceptable, haha.